Makefile

```diff
@@ -85,7 +88,7 @@ all: install lint test
 
 BUILD_TARGETS := build install
 
-build: BUILD_ARGS=-o $(BUILDDIR)/
+ build: BUILD_ARGS=-o $(BUILDDIR)/
```

Comment:
is this leading space right?  I thought make file commands have to have no leading spaces for the definition

Reply:
This is a typo that is fixed now. Thanks for catching that

It worked locally for me. I remember we were having issues in CI when having 4 spaces instead of tabs. That seems to be breaking things